### DISCUSSION TITLE
Settings doesn't retain by player.

### DISCUSSION THREAD

**Discussion by HikaruIchijyo** [anilbeesetti/nextplayer#951] [General]
Idk if it is bug or not
Select a video play it at 2.0 speed 
Go to some other app when u open app again it will reset playback speed to 1.0
Dunno if it is bug or intended behavior that's y asking here

**Comment by anilbeesetti** [comment]
Hi @HikaruIchijyo, It should be retaining the playback speed preference. Can you share a screen recording of the issue if possible

**Reply by HikaruIchijyo** [reply]
Ok i will

**Reply by HikaruIchijyo** [reply]
The easient way is set speed lock phone screen then unlock & it resets to 1.0 while it does play from where it stops

**Reply by HikaruIchijyo** [reply]
> Hi @HikaruIchijyo, It should be retaining the playback speed preference. Can you share a screen recording of the issue if possible

Here is the link fr video

https://storage.tgxbots.workers.dev/?start=KRBOTZ_174340
